packages/powershell/oh-my-posh/oh-my-posh.psm1

```diff
@@ -70,6 +76,9 @@ function Set-PoshPrompt {
         $standardOut
         $global:LASTEXITCODE = $realLASTEXITCODE
         Remove-Variable realLASTEXITCODE -Confirm:$false
+        Start-Job -ScriptBlock {
```

Comment:
I tried but it does not work with Start-Job. It works if the call is done directly like this:
``` 
    Remove-Variable realLASTEXITCODE -Confirm:$false
    Set-GitStatus
```
and performance wise, it's slower than version 2 with the same config.

Reply:
I assume this is on Windows? I noticed slower behavior on Windows with the git segement vs MacOS and Linux. Not sure why.

Reply:
yes on windows. In wsl2 it's really fast, no latency.

Reply:
and in wsl2 I don't have any completion if I hit tab after `git add`

Reply:
> yes on windows. In wsl2 it's really fast, no latency.

Yup. So speeding this up is a Windows only thing. I could optionally add the `get-gitstatus` output as input and parse that for a faster experience. Could be interesting to look at.

Reply:
It would be nice to have method call timings for debugging purpose when invoking segments. Could be turned on with a parameter.

Reply:
The truth is, this isn't an issue anywhere else but Windows. I'm more in favor of calling `Get-GitStatus` and passing the output to oh-my-posh to avoid unneeded calls.